.github/workflows/lint.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
       - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
         with:
-          go-version: '1.21'
+          go-version: '1.22'
```

Comment:
We can use this option: `go-version-file: 'go.mod'`

Reply:
Implemented